mix/ContractCallDataEncoder.h

```diff
@@ -71,11 +71,15 @@ class ContractCallDataEncoder
 	QString toString(bool _b);
 	QString toString(dev::bytes const& _b);
 	bool asString(dev::bytes const& _b, QString& _str);
+	QList<int> extractDimension(QString const& _type);
```

Comment:
If this function is used internally (looks like) do not place is as a member function. Hide it in cpp file.